commit 5009d9ecc692076360ff777e47db9902c7bff669
Author: Brad Hubbard <bhubbard@redhat.com>
Date:   Fri Sep 14 14:22:00 2018 +1000

    Rename "include/assert.h"
    
    This conflicts with the system assert.h so rename and change includes to
    reflect the new name.
    
    Fixes: http://tracker.ceph.com/issues/35682
    
    Signed-off-by: Brad Hubbard <bhubbard@redhat.com>

diff --git a/src/mgr/PyModule.cc b/src/mgr/PyModule.cc
index 90333629a60..cd97f4b5286 100644
--- a/src/mgr/PyModule.cc
+++ b/src/mgr/PyModule.cc
@@ -32,7 +32,7 @@ std::string PyModule::config_prefix = "mgr/";
 // Courtesy of http://stackoverflow.com/questions/1418015/how-to-get-python-exception-text
 #include <boost/python.hpp>
 #include <boost/algorithm/string/predicate.hpp>
-#include "include/assert.h"  // boost clobbers this
+#include "include/ceph_assert.h"  // boost clobbers this
 // decode a Python exception into a string
 std::string handle_pyerror()
 {
